torchci/test/jobUtils.test.ts

```diff
@@ -55,30 +55,44 @@ describe("Test various job utils", () => {
     ).toStrictEqual("Test `run_test.py` is usable without boto3/rockset");
   });
 
-  test("test isSameHeadBranch", () => {
-    expect(isSameHeadBranch("", "")).toEqual(false);
+  test("test isSameAuthor", () => {
+    expect(isSameAuthor("", "")).toEqual(false);
 
-    expect(isSameHeadBranch("mock-branch", "")).toEqual(false);
+    expect(isSameAuthor("mock-branch", "")).toEqual(false);
 
-    expect(isSameHeadBranch("", "mock-branch")).toEqual(false);
+    expect(isSameAuthor("", "mock-branch")).toEqual(false);
 
-    expect(isSameHeadBranch("mock-branch", "mock-branch")).toEqual(true);
+    expect(isSameAuthor("mock-branch", "mock-branch")).toEqual(true);
 
-    expect(isSameHeadBranch("ciflow/trunk/1", "ciflow/trunk/2")).toEqual(false);
+    expect(isSameAuthor("one-branch", "another-branch")).toEqual(false);
 
-    expect(isSameHeadBranch("ciflow/trunk/1", "ciflow/trunk/1")).toEqual(true);
+    expect(isSameAuthor("ciflow/trunk/1", "ciflow/trunk/2")).toEqual(false);
```

Comment:
Does this mean trunk + other ciflow workflows won't be able to tell who the author is?

Reply:
Good catch! You're right.  This needs to be handled differently

Reply:
perhaps we should just query the author from rockset? I think this will also have an issue if people don't follow convention and don't start their branch names with their username

Reply:
Yeah, agree.  That would be a better approach and this quick and dirty workaround.  I will retrofit the PR to get the proper author info.